src/main/java/com/pdffiller/client/utils/MultipartHelper.java

```diff
@@ -0,0 +1,88 @@
+package com.pdffiller.client.utils;
+
+import java.io.File;
+import java.io.IOException;
+import java.math.BigInteger;
+import java.nio.file.Files;
+import java.security.SecureRandom;
+import java.util.HashMap;
+
+public class MultipartHelper {
```

Comment:
Let's call it MultipartMessageBuilder. *Helper names are not popular in Java

Add Logger to this class it will help with message debugging